realm/realm-library/src/main/java/io/realm/internal/TableView.java

```diff
@@ -826,6 +838,7 @@ public long sync() {
     private native long nativeFindAllFloat(long nativePtr, long columnIndex, float value);
     private native long nativeFindAllDouble(long nativePtr, long columnIndex, double value);
     private native long nativeFindAllDate(long nativePtr, long columnIndex, long dateTimeValue);
+    private native long nativeFindBySourceNdx(long nativePtr, long rowIndex);
```

Comment:
This naming is a bit unconventional, using `nativeFind(long nativePtr, long sourceRowIndex)` would be better IMO